index.js

```diff
@@ -156,15 +156,22 @@ app.get("/direct", async (req, res, next) => {
 	data.pipe(res);
 });
 
-async function getIconURL(size, type, name){
+async function getIconURL(size, type, name, skin){
 	size = size.replace(/[^A-Za-z0-9.]/g, "");
 	name = name.replace(/[^A-Za-z0-9.]/g, "");
-	type - type.replace(/[^A-Za-z0-9.]/g, "");
+	type = type.replace(/[^A-Za-z0-9.]/g, "");
+	skin = skin.replace(/[^A-Za-z0-9.]/g, "");
 
 	name = "icons/" + size + "/" + name;
 
 	try {
```

Comment:
It will be nice to step "skin" folder testing step if skin value is empty

Reply:
it won't be empty because the default is "none"

Reply:
And what is the reasoning of having such value?
When no-skin value provided, we need to return image from base set of images, without checking the skin folder ( as we want to preserve compatibility ) and if we are skipping check, using an empty value looks as a better choice ( as string value will not be used anywhere )

Reply:
updated